src/app/core/ui/ui/ui.component.scss

```diff
@@ -65,8 +65,12 @@ body {
   grid-template-rows: auto 1fr;
   height: 100vh;
 
-  /* mobile viewport bug fix */
-  min-height: -webkit-fill-available;
+  /* mobile viewport bug fix. */
+  // The viewport height on iOs is not necessarily the visible screen size.
+  // There might be a bar on the top of the screen. The following line
+  // fixes this, but since this is no official css feature, this line should
+  // be replaced as soon as a nicer solution exists.
+  height: -webkit-fill-available;
```

Comment:
This seems to break the height on normal devices. At least on my computer the navigation bar is far too long and needs to be scrolled

Reply:
And probably related: the "Record Attendance" screen is now too short (i.e. white space at bottom half of screen)

Reply:
and the primary action (blue bubble button on the bottom left) is not fixed but scrolling along other content